### PR TITLE
Change test methods visibility.

### PR DESCRIPTION
**Pull request by ming13**
Tests are passing locally: 187 of 187.

### CODE REVIEW

File: library/tests/src/com/google/maps/android/clustering/QuadItemTest.java [hunk 1]
```diff
@@ -47,10 +47,6 @@ public String getSnippet() {
         }
     }
 
-    public void setUp() {
```
Comment (ming13) [inline library/tests/src/com/google/maps/android/clustering/QuadItemTest.java]: Removed unused `setUp` blocks there and there.

File: library/tests/src/com/google/maps/android/clustering/QuadItemTest.java [hunk 2]
```diff
@@ -24,10 +24,10 @@
 
 public class QuadItemTest {
 
-    public class TestingItem implements ClusterItem {
+    private class TestingItem implements ClusterItem {
```
Comment (ming13) [inline library/tests/src/com/google/maps/android/clustering/QuadItemTest.java]: Made everything fixture-related `private`.

File: library/tests/src/com/google/maps/android/data/geojson/GeoJsonFeatureTest.java [hunk 3]
```diff
@@ -116,6 +118,7 @@ public void testGeometry() {
         Assert.assertEquals(lineString, feature.getGeometry());
     }
 
+    @Test
     public void testGetBoundingBox() {
```
Comment (ming13) [inline library/tests/src/com/google/maps/android/data/geojson/GeoJsonFeatureTest.java]: These three were not annotation-marked so were not running before.

**Comment by ming13** [comment]
@jpoehnelt, resolved. Please squash commits while merging.

**Comment by friederbluemle** [comment]
Hi @ming13 - Would you be merging #560 first and rebasing your PR?

**Comment by friederbluemle** [comment]
Hi @ming13 - So sorry, it looks like #560 created a bunch of conflicts with your branch. I've reapplied the remaining changes from your commit on top of the latest master, you can fetch it from here for your convenience: https://github.com/friederbluemle/android-maps-utils/tree/test-visibility-rebased
Once you've updated your branch, we can merge this PR. 👍

**Comment by friederbluemle** [comment]
@jpoehnelt @ming13 - The changes here still make sense. I created a new PR #569 with the rebased commit (left the author as @ming13). Please check.

**Pull request closed without merging**